app/Controllers/categoryController.php

```diff
@@ -80,45 +80,80 @@ public function createAction() :void {
 
 	/**
 	 * This action updates the given category.
-	 * @todo Check whether this function is used at all
-	 * @see FreshRSS_subscription_Controller::categoryAction() (consider merging)
```

Comment:
Merged with `FreshRSS_subscription_Controller::categoryAction()` (to avoid partially duplicated and dead code)